lib/deep-equal.test.js

```diff
@@ -800,4 +809,14 @@ describe("deepEqual", function() {
             assert.isFalse(checkDeep);
         });
     });
+
+    describe("when called with matchers", function() {
```

Comment:
Double test

Reply:
I will remove this in favor of your test 👍